docs/b2b-edition/specs/storefront/storefront/sales-rep.yaml

```diff
@@ -137,7 +137,7 @@ paths:
                     data: {}
       description: |-
         Update status for when a super admin masquerades as a company by userId and companyId.
-        <br>Equivalent Storefront GraphQL API Mutation: `superAdminEndMasquerade`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).
+        <br>Equivalent Storefront GraphQL API Mutation: `superAdminEndMasquerade`. For more information, see the [GraphQL Playground](https://api-b2b.bigcommerce.com/graphql/playground).</br>
```

Comment:
Does our linter add in these `</br>` tags? I ask because the Dev Center doesn't support break tags in general. So we should instead be removing `<br>` tags instead of closing them.

Reply:
@becomevocal I am not sure how the tags were added initially. But the tags break the line correctly and display the text similar to stoplight when I close the tags. I am assuming this was the desired intent.

Reply:
OK, I can remove them.

Reply:
Yes, I was previewing them in a swagger editor.